fixtures/net/ip_vs

```diff
@@ -12,3 +12,7 @@ TCP  C0A80039:0CEA wlc
 TCP  C0A80037:0CEA wlc  
   -> C0A8321A:0CEA      Tunnel  0      0          0         
   -> C0A83120:0CEA      Tunnel  100    0          0         
+TCP  [2620:0000:0000:0000:0000:0000:0000:0001]:0050 sh 
```

Comment:
Looks like IPv4 is straight-up hexadecimal but IPv6 is represented with colons. That the original parsing _would_ have handled straight-hexadecimal IPv6 addresses was incidental should not be construed as these actually being part of the format.